common/src/main/templates/io/netty/util/collection/KObjectMap.template

```diff
@@ -76,48 +71,14 @@ public interface @K@ObjectMap<V> {
     V remove(@k@ key);
 
     /**
-     * Returns the number of entries contained in this map.
-     */
-    int size();
-
-    /**
-     * Indicates whether or not this map is empty (i.e {@link #size()} == {@code 0]).
-
+     * Gets an iterable collection of primitive entries contained in this map. As an optimization,
```

Comment:
@nmittler - Can we leave the `collection` terminology out of this comment? Maybe something like...

```
Get an iterator to traverse over the primitive entries contained in this map.
```